webcam.go

```diff
@@ -106,6 +106,26 @@ func (w *Webcam) GetSupportedFrameSizes(f PixelFormat) []FrameSize {
 	return result
 }
 
+// GetSupportedFramerates returns supported frame rates for a given image format and frame size.
+func (w *Webcam) GetSupportedFramerates(fp PixelFormat, width uint32, height uint32) []FrameRate {
+	var result []FrameRate
+	var index uint32
+	var err error
+
+	// keep incrementing the index value until we get an EINVAL error
+	index = 0
+	for err == nil {
```

Comment:
So I'm seeing 
> Only for the V4L2_FRMIVAL_TYPE_DISCRETE type does it make sense to increase the index value to receive more frame intervals.

It sounds like this means we should only loop if `_type == V4L2_FRMIVAL_TYPE_DISCRETE`. I see for the frame sizes function it does the same thing. I guess it doesn't matter though if the other types just return `EINVAL`?

Reply:
Yes, if index 0 returns Discrete we could check for this and break out. However, the err check still works in this scenario.